src/LibraryManager/Providers/FileSystem/FileSystemProvider.cs

```diff
@@ -86,6 +86,11 @@ public ILibraryCatalog GetCatalog()
                         return LibraryInstallationResult.FromCancelled(desiredState);
                     }
 
+                    if (string.IsNullOrEmpty(file))
+                    {
+                        return new LibraryInstallationResult(desiredState, PredefinedErrors.CouldNotWriteFile(file));
```

Comment:
See comment above - I think this would look a bit strange with missing file name.